src/NexusMods.App.UI/Windows/MainWindow.axaml.cs

```diff
@@ -65,6 +65,13 @@ public MainWindow()
             this.WhenAnyValue(view => view.IsActive)
                 .BindTo(this, view => view.ViewModel!.IsActive)
                 .DisposeWith(disposables);
+
+            ViewModel!.ActivateWindowCommand
+                .OnUI()
+                .Subscribe(_ =>
+            {
+                Activate();
```

Comment:
I can't really test this on Linux, but have you tried some alternatives like `Focus()` and `Topmost`?

Reply:
On windows it works if the window is simply behind the browser, but if the window is minimized nothing happens.

Reply:
```suggestion
                if (WindowState == WindowState.Minimized)
                {
                    WindowState = WindowState.Normal;
                }
                Activate();
```
The above should bring restore the window if it was minimized. 
I tested this on Windows.